app/src/main/kotlin/com/wire/android/ui/authentication/create/common/CreateAccountBaseViewModel.kt

```diff
@@ -291,6 +294,12 @@ private fun RegisterClientResult.Failure.toCodeError() = when (this) {
     RegisterClientResult.Failure.TooManyClients -> CreateAccountCodeViewState.CodeError.TooManyDevicesError
     RegisterClientResult.Failure.InvalidCredentials -> CreateAccountCodeViewState.CodeError.DialogError.InvalidEmailError
     is RegisterClientResult.Failure.Generic -> CreateAccountCodeViewState.CodeError.DialogError.GenericError(this.genericFailure)
+    RegisterClientResult.Failure.PasswordAuthRequired -> {
+        Log.wtf("RegisterClient", "wrong password when register client after creating a new account")
```

Comment:
:D do we need that anymore?

Reply:
The error type or the Log?
I wanted to leave the log here because it's an error that should never happen, so if it did happen we need to know.

Reply:
Can we use appLogger instead?